web-ui/src/pages/SkillReportPage.jsx

```diff
@@ -40,9 +41,10 @@ const SkillReportPage = props => {
       memberIds.includes(memberSkill.id)
     );
     if (memberSkillsFound && memberIds) {
-      setSearchResults(memberSkillsFound);
+      let newSort = sortMembersBySkill(memberSkillsFound);
+      setSearchResults(newSort);
```

Comment:
Is it possible to assign `newSort` as a `const`, or does setSearchResults mutate the array?

Reply:
I favor let's over const's in general within functions.

Reply:
Can you explain why you favor using `let` to hold the result of a  function call?
I think the general recommendation is to use `const` everywhere possible and it's possible here.

Reply:
I subscribe more to this train of thought, which is that consts are best used for module scope. ‘Let’ gives the option to have flexibility should you need it while programming, so I tend to use it in a block scoped manner. I also like that it is block scoped, so I don’t have to worry about it outside of my function.

I know this is an old blog post, but to an extant I think the argument made in favor of “let” for block scoped variables is still relevant.
https://madhatted.com/2016/1/25/let-it-be